### ISSUE TITLE
Instance norm instead of Batch norm

### ISSUE THREAD

**Issue by KimberleyJensen** [junyuchen-cjy/DTTNet-Pytorch#1]
Hi @junyuchen-cjy 

In kuielabs tfc_tdf_v3 they replaced BN+ReLU with IN+GELU but reading your code you went with BN, did you do any testing with IN? As using IN is preferred for me as I have to train with smaller batch sizes.

Thanks 😃

**Comment by junyuchen-cjy** [comment]
Thank you for the first issue. Certainly, you can explore using IN+GELU+CONV in place of Conv+BN+ReLU. I've taken a more cautious approach because of the time constraints that prevent me from conducting experiments. I'm eagerly anticipating the outcome of these changes.
